### PR TITLE
#2291 added in Publish onto the icon on Admin/Campaign

### PR DESCRIPTION
**Pull request by djhmateer**

### CODE REVIEW

**Comment by stevejgordon** [comment]
@djhmateer - One thing I didn't press during the codeathon as I didn't want to push too much to people too fast was linking the PR to the issue. If you put something like Fixes #2291 in the PR comment it will link it to the original issue. This is great for reviewers as it automatically adds a hyperlink and also, once the PR is merged, the issue is auto-closed.

File: AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Campaign/Index.cshtml [hunk 1]
```diff
@@ -40,7 +40,7 @@
                         @if (!item.Published)
                         {
                             <div class="btn-group btn-group-sm row-btn-margin">
-                                <a data-toggle="tooltip" data-placement="top" title="Publish" asp-controller="Campaign" asp-action="Publish" asp-area="@AreaNames.Admin" asp-route-id="@item.Id" class="btn btn-primary"><span class="fa fa-upload"></span></a>
+                                <a data-toggle="tooltip" data-placement="top" title="Publish" asp-controller="Campaign" asp-action="Publish" asp-area="@AreaNames.Admin" asp-route-id="@item.Id" class="btn btn-primary"><span class="fa fa-upload"></span> Publish</a>
```
Comment (stevejgordon) [inline AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Campaign/Index.cshtml]: I don't think the icon is helping anyone. Could you remove the <span class="fa fa-upload"></span> part so it's just the text label instead please?

**Comment by stevejgordon** [comment]
@djhmateer Thanks for this PR. A subtle change if you wouldn't mind. Thanks!

**Comment by MisterJames** [comment]
@djhmateer Just a small issue if you wouldn't mind addressing so we can merge? (See Steve's comment)

**Comment by MisterJames** [comment]
We'll leave this one as-is and pull out the icon if recommended after the rest of the UX review.

**Pull request merged**